person.rb

```diff
@@ -0,0 +1,37 @@
+# Class containing data about a person
+require_relative 'nameable'
+require './rental'
+
+class Person < nameable
+  # Fixed the class name to inherit from Nameable (capitalized)
+  attr_accessor :name, :age
```

Comment:
- [ ] Getters and Setters for @rentals should be present

Reply:
getters and setters added for rental.

attr_accessor :rentals